install/gcp/deployment_manager/istio-cluster.jinja.display

```diff
@@ -112,6 +112,22 @@ input:
       section: MONITORING
       boolean_group: MONITORING_GROUP
 
+    - name: enableKiali
+      title: Enable Kiali for deployment visualization
+      tooltip: Enable Kiali on the cluster
+      section: MONITORING
+      boolean_group: MONITORING_GROUP
+
+    - name: kialiUsername
+      title: Default Admin username for Kiali
+      tooltip: Default Admin username for Kiali
```

Comment:
in Kiali, what happens when this is not specified?  A null entry?  Or does GCP DM not permit this?  Seems like you would want this to be a "non-optional setting of nonzero".  Same with kialiPassword.

Reply:
added in a minimum length value for `kialiPassword` into the `schema` file so the template will complain if someone overrides it (i could also devise [dm schema regex patters](https://cloud.google.com/deployment-manager/docs/configuration/templates/using-schemas#properties) but i can't generate a random one ondemand.

```
  kialiPassword:
    type: string
    description: Default username for Kiali
    minLength: 8
    default: mysecret
```